R/RcppExports.R

```diff
@@ -1,7 +1,7 @@
 # Generated by using Rcpp::compileAttributes() -> do not edit by hand
 # Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
-.deserialize_json <- function(json, query = NULL, empty_array = NULL, empty_object = NULL, single_null = NULL, parse_error_ok = FALSE, on_parse_error = NULL, query_error_ok = FALSE, on_query_error = NULL, simplify_to = 0L, type_policy = 0L, int64_r_type = 0L) {
+.deserialize_json <- function(json, query = NULL, empty_array = NULL, empty_object = NULL, single_null = NULL, parse_error_ok = FALSE, on_parse_error = NULL, query_error_ok = FALSE, on_query_error = NULL, simplify_to = 3L, type_policy = 0L, int64_r_type = 0L) {
```

Comment:
I am not sure I am in favour of sudden and hidden behavior changes.

Maybe (and that is a big maybe) we can add a new user-facing function which defaults to calling with `simplify_to=3L`.

Reply:
As it is not currently exported, there is no behavioural change. `fparse()` always calls it with the arguments defined.